RPGCalendar/RPGCalendar/ClientApp/src/app/game-list/game-list.component.ts

```diff
@@ -8,8 +8,66 @@ import { Router } from  '@angular/router';
   styleUrls: ['./game-list.component.css']
 })
 export class GameListComponent implements OnInit {
+  headers: any;
 
-  constructor(private router: Router) { }
+  rows: any;
+
+  aheaders: any;
+
+  arows: any;
+
+  condition: any;
+  
+  constructor(private router: Router) {
+    this.condition ={
+      availGames : false
+    };
+
+    this.headers = ['Title', 'Game Master', 'Num. of Players'];
+
+    this.rows = [
+      {
+        "Title" : "D&D",
+        "Game Master" : "Kalipso",
+        "Num. of Players" : "4",
+      },
+      {
+        "Title" : "GloomHaven",
+        "Game Master" : "Clifficus",
+        "Num. of Players" : "3",
+      },
+      {
+        "Title" : "Mice & Mystics",
+        "Game Master" : "Ni",
+        "Num. of Players" : "5",
+      }
+    ];
+    
+    this.aheaders = ['Title', 'Game Master', 'Select'];
+
+    this.arows = [
+      {
+        "Title" : "Possible Game 1", 
+        "Game Master" : "Random Dude",
+        "Select" : '<input type="checkbox">'
+      },
+      {
+        "Title" : "Possible Game 2", 
+        "Game Master" : "Random Dudett",
+        "Select" : "<input type='checkbox'>"
+      },
+      {
+        "Title" : "Possible Game 3", 
+        "Game Master" : "Random Persona",
+        "Select" : "<input type='checkbox'>"
+      }
+    ];
+
+    function deleteRow() {
+      document.getElementById("games").deleteRow(0);
```

Comment:
This line is throwing an error on your build. 

`ERROR in app/game-list/game-list.component.ts:67:40 - error TS2339: Property 'deleteRow' does not exist on type 'HTMLElement'.
  
  67       document.getElementById("games").deleteRow(0);
`